src/components/MDX/SandpackWithHTMLOutput.tsx

```diff
@@ -49,8 +49,8 @@ export default function formatHTML(markup) {
 const packageJSON = `
 {
   "dependencies": {
-    "react": "18.3.0-canary-6db7f4209-20231021",
```

Comment:
verifying for `SandpackWithHTMLOutput` instances:
- `/reference/react-dom/components/link#linking-to-related-resources` ✅ 
- `/reference/react-dom/components/link#linking-to-a-stylesheet` ✅ 
- `/reference/react-dom/components/link#controlling-stylesheet-precedence` ✅ 
- `/reference/react-dom/components/link#deduplicated-stylesheet-rendering` ✅ 
- `/reference/react-dom/components/meta#annotating-the-document-with-metadata` ✅ 
- `/reference/react-dom/components/script#rendering-an-external-script` ✅ 
- `/reference/react-dom/components/script#rendering-an-inline-script`  ❌ 
```diff
-      <script>ga(&#x27;send&#x27;, &#x27;pageview&#x27;);</script>
+      <script>ga('send', 'pageview');</script>
```
- `/reference/react-dom/components/style` ❌ 
```diff
+ data-href="PieChart-[&quot;red&quot;,&quot;green&quot;,&quot;blue&quot;]">#:R2: .color-0: { color: "red"; },#:R2: .color-1: { color: "green"; },#:R2: .color-2: { color: "blue"; }
- data-href="PieChart-[&quot;red&quot;,&quot;green&quot;,&quot;blue&quot;]">#:R2: .color-0: { color: &quot;red&quot;; },#:R2: .color-1: { color: &quot;green&quot;; },#:R2: .color-2: { color: &quot;blue&quot;; }
```
- /reference/react-dom/components/title#set-the-document-title ✅